### PR TITLE
Allow fallback when no package.json

### PR DESCRIPTION
**Pull request by phillip-kruger**
This PR adds a fallback when package.json is not found. 
That makes older structure mvnpm jars work

It also:
- Update the Quarkus version
- Change to use NIO 
- Remove the qute dependency (not really needed)

### CODE REVIEW

**Comment by ia3andy** [comment]
I don't see where you create the package.json (We have some code to do it in the history)
Without it, it won't work?

**Review by edewit — APPROVED** [review]
:+1:

File: src/main/java/io/mvnpm/esbuild/model/AutoEntryPoint.java [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-import java.util.Map;
 import java.util.Objects;
 import java.util.Set;
 
 import static io.mvnpm.esbuild.util.Copy.copyEntries;
+import java.io.StringWriter;
+import java.io.UncheckedIOException;
 
 public class AutoEntryPoint implements EntryPoint {
     private static final Set<String> SCRIPTS = Set.of("js", "ts", "jsx", "tsx");
```
Comment (ia3andy) [inline src/main/java/io/mvnpm/esbuild/model/AutoEntryPoint.java:16]: @phillip-kruger 
We need to add those before releasing: (not related to your PR)
```java
Set.of("js", "ts", "jsx", "tsx", "mjs", "mts", "cjs", "cts");
```

Comment (phillip-kruger) [reply src/main/java/io/mvnpm/esbuild/model/AutoEntryPoint.java:16]: OK do you want to do a PR ?

**Pull request merged**